dicomexporter/exporter.py

```diff
@@ -13,6 +13,31 @@
 # from helpers.itk import convertITKTypeToVTKType, getMetadata, getMetadataList
 # from helpers.volume import VolumeData
 
+class IterableEnum(type):
+    def __iter__(self):
+        for attr in dir(self):
+            if not attr.startswith('__'):
+                yield attr
+
+
+class ALLOWED_EXTENSIONS(metaclass=IterableEnum):
+    vti = 'vti'
+    vtkjs = 'vtkjs'
+
+
+def extractExtensionsFromFilePath(path):
+    """
+    Given a path, return it without extensions and a list of all its
+    extensions (without a leading dot '.').
+    """
+    path_without_extensions = os.path.join(
+        os.path.dirname(path), os.path.basename(path).split(os.extsep)[0]
+    )
+    extensions = os.path.basename(path).split(os.extsep)[1:]
+    if len(extensions) == 1:

```

Comment:
I'm not sure that you need to manage this case. I think that it would be simplier to always return a tuple of string/array.
In the function description, you can just define the output as a tuple of string and array.
The array contains the extracted extensions in the order of there are defined.
It's up to the user to manage the extensions as he wants.

Reply:
Since I am unsure what it will look like to handle files with multiple extensions, I take the last extension. Should be fine for now until we actually need to handle multiple extensions. What do you think?

Reply:
I think that it would be better to have string/array.
I could add some test to check this method with all the cases you think about.
Then we'll update it if users are not familiar with the current working.
@finetjul , WDYT ?

Reply:
Let's keep it simple if possible for now (no funding for tests as of now)

Reply:
@laurennlam I applied the changes as you wanted: extractExtensionsFromFilePath now always returns a tuple containing a string (the path without extensions) and a list (the extensions).
And I just take the last element of the return extension list and treat it as the actual extension for now.